lexers/j/jungle.go

```diff
@@ -0,0 +1,50 @@
+package j
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var MonkeyC = internal.Register(MustNewLexer(
```

Comment:
This should be called Jungle shouldn't it?

Reply:
good catch, copy/paste typo. Will fix these and clean up the history